### DISCUSSION TITLE
What are the system requirements for running DemoGPT?

### DISCUSSION THREAD

**Discussion by kochhar-mw** [melih-unsal/DemoGPT#2] [Q&A] [answered]
Could you shed some light on the model which is used and the system resources needed to run it? Would this run on a MacBook or does it need GPU resources?

**Comment by melih-unsal** [comment, marked as answer]
Hi @kochhar-mw thanks for your attention. The system is based on GPT 3.5-turbo API usage so you only need to have OPENAI_API_KEY to run it but no GPU resource. The system fundamentally needs LangChain and Streamlit libraries. If you encounter any further issues, don't hesitate to report them. Your feedback is highly appreciated and helps us improve the overall user experience.
Best,
Melih
